Pod/Classes/Logging/Log.swift

```diff
@@ -49,11 +53,38 @@ open class Log {
         }
     }
 
+    public static private(set) var standard = Log("", logLevel: .off)
+
+    /// Static instance used for helper methods.
+    open class var instance: Log {
+        return standard
+    }
```

Comment:
I think this is possibly still confusing for people using the old 

`class NetworkLog: Log {}` approach, because `NetworkLog.instance` == `Log.instance` which means that `NetworkLog.logLevel = .off` still manipulates `Log.logLevel`.

I don't know if there's a good alternative approach here because there is no class var storage, so at a certain point you'll always have to reach for a static instance that is shared among all subclasses. So maybe it's best to just not even have any `open` override points that could potentially point to the same static instance, and just make it extra clear that setting `Log.logLevel` or `NetworkLog.logLevel` sets it for all instances and subclasses since they all refer to the same static variable.

Would also maybe suggest naming this singleton `shared`

Reply:
That's fair, I was attempting to still allow for a similar `class NetworkLog: Log {}` approach, but giving a way to override which instance gets used from the static functions. Where I ended up, is a little more verbose.

```
    class NetworkLog: Log {
        static var networkLog = Log("Network")

        override class var instance: Log {
            return networkLog
        }
    }
```

Reply:
Ah yeah. In that case it would be nice if there was a way to force "holding it properly", and enforce this override for subclasses, but that isn't really possible either

Reply:
I haven't found a good way to do that yet :-/